github/activity_events.go

```diff
@@ -30,7 +30,21 @@ func (e Event) String() string {
 
 // Payload returns the parsed event payload. For recognized event types,
 // a value of the corresponding struct type will be returned.
+//
+// Deprecated: Use ParsePayload instead, which returns an error
+// rather than panics if JSON unmarshaling raw payload fails.
 func (e *Event) Payload() (payload interface{}) {
+	var err error
+	payload, err = e.ParsePayload()
+	if err != nil {
+		panic(err)
+	}
+	return payload
+}
```

Comment:
Really minor, but would you mind swapping the order of `Payload` and `ParsePayload` in the source? I think the deprecated func should be below the preferred one.